apps/client/src/common/components/multi-part-progress-bar/MultiPartProgressBar.scss

```diff
@@ -7,42 +7,51 @@ $progress-bar-br: 3px;
   position: relative;
   width: 100%;
   height: $progress-bar-size;
-  border-radius: $progress-bar-br;
-  background-color: var(--timer-progress-bg-override, $viewer-card-bg-color);
-  display: flex;
-  overflow: hidden;
 
-  &--hidden {
+  &.hidden {
```

Comment:
question: is there a reason for this name change?
the CSS properties that we expose to client use the BEM convention.

Changing it will break users setup and our convention

Reply:
Nope I think I was just matching with what @alex-Arc wrote above but will switch both to BEM style